src/webapp/flask_common.py

```diff
@@ -0,0 +1,22 @@
+from flask import Response
+from .common import to_csv, to_json_bytes
+
+
+def create_accepted_response(data: list, headers, default=None) -> Response:
```

Comment:
```suggestion
from typing import List

def create_accepted_response(data: List[str], headers, default=None) -> Response:
```
If it's a list of strings you should put that in the annotation. Note that the webapp uses Python 3.6 which means you can't use the lowercase `list` for a type hint (that was added in [3.9](https://peps.python.org/pep-0585/) )